helm_deploy/send-legal-mail-to-prisons/values.yaml

```diff
@@ -93,11 +93,15 @@ gotenberg:
   securityContext:
     runAsUser: 1001
     runAsNonRoot: true
+    readOnlyRootFilesystem: false
+    allowPrivilegeEscalation: false
+    privileged: false
 
   env:
     GOOGLE_CHROME_IGNORE_CERTIFICATE_ERRORS: 1
     DISABLE_UNOCONV: 1
     DEFAULT_WAIT_TIMEOUT: 30
+    # --chromium-ignore-certificate-errors=true
```

Comment:
Comment is for the option that would replace the env, might not be required though.